src/webots/vrml/WbProtoManager.hpp

```diff
@@ -34,21 +34,25 @@ class WbVersion;
 
 class WbExternProto {
 public:
-  WbExternProto(const QString &name, const QString &url, bool isImportable) :
+  WbExternProto(const QString &name, const QString &url, bool isImportable, bool isFromRootNodeConversion = false) :
     mName(name),
     mUrl(url),
-    mImportable(isImportable) {}
+    mImportable(isImportable),
+    mFromRootNodeConversion(isFromRootNodeConversion) {}
 
   const QString &name() const { return mName; }
   void setUrl(const QString &url) { mUrl = url; }
   const QString &url() const { return mUrl; }
   bool isImportable() const { return mImportable; }
   void setImportable(bool value) { mImportable = value; }
+  bool isFromRootNodeConversion() const { return mFromRootNodeConversion; }
+  void setFromRootNodeConversion(bool value) { mFromRootNodeConversion = value; }
```

Comment:
Since it is always called with a `false` parameter, should probably remove the parameter and rename the method.